src/cockpit/389-console/src/lib/plugins/retroChangelog.jsx

```diff
@@ -191,12 +288,29 @@ class RetroChangelog extends React.Component {
             this.state.isReplicated ? "TRUE" : "FALSE",
             "--max-age",
             maxAge || "delete",
-            "--exclude-suffix",
-            this.state.excludeSuffix || "delete",
             "--trim-interval",
             this.state.trimInterval.toString() || "300"
         ];
-
+        if (this.state._excludeSuffix != this.state.excludeSuffix) {
+            cmd = [...cmd, "--exclude-suffix"];
+            if (this.state.excludeSuffix.length != 0) {
+                for (const value of this.state.excludeSuffix) {
+                    cmd = [...cmd, value];
+                }
+            } else {
+                cmd = [...cmd, "delete"];
+            }
+        }
```

Comment:
If I try to add a second value here, - it adds an empty item and complains about invalid DN (even if it's valid):

![image](https://user-images.githubusercontent.com/5634892/147165349-bc294c4e-ab99-4485-9760-44d1cd237413.png)